src/cmd/setup-tachyon.js

```diff
@@ -648,7 +696,8 @@ module.exports = class SetupTachyonCommands extends CLICommandBase {
 			'systemPassword',
 			'productId',
 			'timezone',
-			'wifi'
+			'wifi',
+			'country',
```

Comment:
I'm not seeing this at first glane but does loading config from JSON load the country too?

Reply:
right if there is no country into the json, then it will get the default. this part is when you saving the config file